File: scripts/commands/generic/spatialChatInternal.py
from intents.chat import SpatialChatIntent
import sys

#Args: unkInt chatType moodId unkInt2 langId message
def execute(objManager, player, target, args):
	#print ('unkInt=' + cmdArgs[0] + ' chatType=' + cmdArgs[1] + ' moodId=' + cmdArgs[2] + ' unkInt2=' + cmdArgs[3] + ' langId=' + cmdArgs[4])

	cmdArgs = args.split(' ', 5)
	SpatialChatIntent(player, int(cmdArgs[1]), cmdArgs[5]).broadcast()
	return

File: scripts/commands/generic/spatialChatInternal.js
function executeCommand(galacticManager, player, target, args) {
	var cmdArgs = args.split(' ', 5);
	var SpatialChatIntent = Java.type("intents.chat.SpatialChatIntent");
    new SpatialChatIntent(player, cmdArgs[1], args.substring(10), cmdArgs[2]).broadcast();
}